LambdaEngine/Source/Rendering/ParticleManager.cpp

```diff
@@ -17,65 +17,69 @@ namespace LambdaEngine
 {
 	void ParticleManager::Init(uint32 maxParticleCapacity, ASBuilder* pASBuilder)
 	{
+		if (!m_Initilized)
+		{
+			m_Initilized = true;
```

Comment:
spelling: ~m_Initilized~ -> m_Initialized

I would also set `m_Initialized` to true after you have run all necessary steps not before.

Reply:
Good spot!